sdk/communication/communication-administration/src/common/logger.ts

```diff
@@ -4,6 +4,6 @@
 import { createClientLogger } from "@azure/logger";
 
 /**
- * The @azure/logger configuration for this package.
+ * The \@azure/logger configuration for this package.
```

Comment:
I think this should be fine too:
```suggestion
 * The `@azure/logger` configuration for this package.
```